splunk_connect_for_snmp_poller/manager/tasks.py

```diff
@@ -72,98 +147,23 @@ def snmp_polling(
-                            try:
-                                mib_string_handler(
-                                    snmp_engine,
-                                    auth_data,
-                                    context_data,
-                                    host,
-                                    port,
-                                    varbind,
-                                    mib_server_url,
-                                    index,
-                                    otel_logs_url,
-                                    otel_metrics_url,
-                                    one_time_flag,
-                                )
-                            except Exception as e:
-                                logger.error(
-                                    f"Error happend while calling mib_string_handler(): {e}"
-                                )
-                        else:
-                            # Perform SNMP polling for oid
-                            try:
-                                if varbind[-1] == "*":
-                                    walk_handler(
-                                        snmp_engine,
-                                        auth_data,
-                                        context_data,
-                                        host,
-                                        port,
-                                        varbind,
-                                        mib_server_url,
-                                        index,
-                                        otel_logs_url,
-                                        otel_metrics_url,
-                                        one_time_flag,
-                                    )
-                                else:
-                                    get_handler(
-                                        snmp_engine,
-                                        auth_data,
-                                        context_data,
-                                        host,
-                                        port,
-                                        varbind,
-                                        mib_server_url,
-                                        index,
-                                        otel_logs_url,
-                                        otel_metrics_url,
-                                        one_time_flag,
-                                    )
-                            except Exception as e:
-                                logger.error(
-                                    f"Invalid format for oid. Error message: {e}"
-                                )
+                # Divide varBinds for WALK/BULK actions
+                varbind_collection = sort_varbinds(varBinds)
+                logger.info(f"Varbind collection: {varbind_collection}")
+                # Perform SNMP BULK
+                get_data(varbind_collection.bulk, bulk_handler, *static_parameters)
```

Comment:
Change this name please. Maybe get_snmp_data? get_data looks like it's performing only an SNMP GET data

Reply:
Done